README.md

```diff
@@ -1,6 +1,6 @@
-# Urban Airship React Native
+# c
```

Comment:
??
```suggestion
# Airship React Native
```

Reply:
my command key is failing on my laptop. I probably tried to copy something